client-templates/bitbucket-server/accept.json.sample

```diff
@@ -628,18 +637,18 @@
     {
       "//": "used to create a Code Insights report",
       "method": "PUT",
-      "path": "/rest/insights/1.0/projects/:project/repos/:repo/commits/:sha/reports/:report",
+      "path": "/rest/insights/latest/projects/:project/repos/:repo/commits/:sha/reports/:report",
       "origin": "https://${BITBUCKET}",
       "auth": {
-      "scheme": "basic",
-      "username": "${BITBUCKET_USERNAME}",
-      "password": "${BITBUCKET_PASSWORD}"
+        "scheme": "basic",
+        "username": "${BITBUCKET_USERNAME}",
+        "password": "${BITBUCKET_PASSWORD}"
       }
     },
     {
       "//": "used to remove a Code Insights report",
       "method": "DELETE",
-      "path": "/rest/insights/1.0/projects/:project/repos/:repo/commits/:sha/reports/:report",
+      "path": "/rest/insights/latest/projects/:project/repos/:repo/commits/:sha/reports/:report",
```

Comment:
@mikomraz 
Please verify if this is correct
Here is link to BBS agent:
https://github.com/snyk/bitbucket-server-agent/blob/1454badf76fa7f12bb8d80760bd2a396c047606d/lib/bitbucket-server/codeinsights.js#L128